### DISCUSSION TITLE
How can i get a wss port ?

### DISCUSSION THREAD

**Discussion by ye-gz-1996** [dogecoin/dogecoin#3100] [Help needed] [answered]
I am running a fullnode, but i dont know how get my own wss port or http port web3 programming, such as Web3.HTTPProvider('https://bsc-dataseed1.defibit.io/') in bsc. Who can help me?

**Comment by patricklodder** [comment, marked as answer]
Unfortunately, there is no websocket port for Dogecoin and there is no support for web3.js - what were you intending to do?

**Reply by ye-gz-1996** [reply]
Sorry sir, I was deceived. I thought DogeChain  was a new chain officially released.Thanks again for your answer！

**Reply by abosaleh82** [reply]
> Unfortunately, there is no websocket port for Dogecoin and there is no support for web3.js - what were you intending to do?

can you help me to get my dogecoin for 2m dogecoin if we success contact me on my email : 
basameer92@gmail.com

**Reply by LookRareNFTs** [reply]
Are there any plans to add WSS support?

**Reply by patricklodder** [reply]
[ElectrumX](https://github.com/spesmilo/electrumx) provides one. [Read the docs](https://electrumx.readthedocs.io/en/latest/), after which you can experiment with my public server at `wss://mainnet.electrum.plddr.eu:50004` before you run your own. You'll need a server with ~12GB RAM to run both a dogecoind (4GB) and this software (8GB).

**Comment by ye-gz-1996** [comment]
Network Name: DogeChain 
Chain ID: 2000
dogechain.dog has nothing to do with actual dogecoin or dogechain.info.
i was cheated.

**Reply by patricklodder** [reply]
I'm sorry that you were deceived in believing this - from what I gather it's a bridged chain but I haven't seen any indication that this bridge and chain is safe. Please be careful.
